client/my-sites/marketing/connections/services-group.jsx

```diff
@@ -103,7 +103,7 @@ const SharingServicesGroup = ( {
 						status="is-warning"
 						showDismiss={ false }
 						text={ translate(
-							'Connect to your social media accounts to enable sharing posts with Jetpack Social.'
+							'You have to enable Jetpack Social to be able to connect your social media accounts and share your posts automatically.'
```

Comment:
I felt the copy could be simplified a bit to make it easier to skim - feel free to ignore though!

```suggestion
							'Please enable Jetpack Social to connect your social media accounts and share your posts automatically.'
```

Reply:
This seems a good improvement. What do you think @gmjuhasz?